internal/builtinfunctions/functions.go

```diff
@@ -83,9 +83,9 @@ func getFloatToIntFunction() schema.CallableFunction {
 		schema.NewDisplayValue(
 			schema.PointerTo("floatToInt"),
 			schema.PointerTo(
-				"Converts a float type into an integer type by down-casting. "+
-					"The value loses any data after the decimal point.\n"+
-					"For example, `5.5` becomes `5`",
+				"Converts a float type into an integer type by discarding the fraction."+
+					" In other words, it is rounded to the nearest integer towards zero.\n"+
```

Comment:
Switching from "discarding the fraction" to "rounded" doesn't seem helpful, since "round" has other connotations. Maybe "In other words, it is truncated towards zero to the nearest integer"?

Reply:
What other connotations could cause confusion here?